ratelimits/limiter_test.go

```diff
@@ -583,7 +599,7 @@ func TestRateLimitError(t *testing.T) {
 				test.AssertNotError(t, err, "expected no error")
 			} else {
 				test.AssertError(t, err, "expected an error")
-				test.AssertContains(t, err.Error(), tc.expectedErr)
+				test.AssertEquals(t, err.Error(), tc.expectedErr)
```

Comment:
nit: I don't love using AssertEquals for error strings, it makes us a bit _too_ precise imo. I'd prefer that the test case have two fields `expectedErr` and `expectedLink`, and assert that the resulting message contains both of them. But that's a super minor thing.